### ISSUE TITLE
New footer on Instructor page

### ISSUE THREAD

**Issue by McD** [Get-Me-Fit/getmefit#93]
Let's remove the FAQs text on instructor page footer https://getmefit.co/mark

Replace with:
LEFT SIDE
Are you a fitness instructor that would like to run classes online? You can set up a FREE Get Me Fit profile in just 5 minutes [link "set up a FREE Get Me Fit profile" to homepage]

RIGHT SIDE
Class participants - Not sure how to get started? Read our help Guide Here [link to http://bit.ly/GMF-Participant-Help]

**Comment by krishna-acondy** [comment]
@McD / @siebevd I've now changed the footer to look like this:

![image](https://user-images.githubusercontent.com/2980428/79070830-09579d00-7cd0-11ea-9f0f-6e354142ec52.png)

There's just one open question - At the moment there's some logic in the footer that shows the left side only when the user is signed in as a member. I've changed that to make it show regardless. Is that okay, or is there some other reason to have that logic in place? (Changes in https://github.com/Get-Me-Fit/getmefit/pull/104).
